quinn/src/platform/unix.rs

```diff
@@ -20,6 +21,8 @@ type IpTosTy = libc::c_int;
 
 impl super::UdpExt for UdpSocket {
     fn init_ext(&self) -> io::Result<()> {
+        Self::caps();
+
```

Comment:
What's this doing here?

Reply:
good catch! I added this just for testing to get it to be actually used and get a debug output. Will remove